### ISSUE TITLE
Is there a way to have multiple instances of the same option?

### ISSUE THREAD

**Issue by ericnewton76** [commandlineparser/commandline#150]
<a href="https://github.com/cdmihai"><img src="https://avatars0.githubusercontent.com/u/2255729?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [cdmihai](https://github.com/cdmihai)**
_Friday Jul 21, 2017 at 00:05 GMT_
_Originally opened as https://github.com/gsscoder/commandline/issues/459_

----

Is there anyway I can express `myapp -o a/b/c -o d/e/f -o g/h/i`?

Ideally it could be expressed like:
```c#
[Option('o', MultipleInstance = true)]
IEnumerable<string> OutputPaths {get; set;}
```

**Comment by ericnewton76** [comment]
<a href="https://github.com/nemec"><img src="https://avatars3.githubusercontent.com/u/136378?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nemec](https://github.com/nemec)**
_Friday Jul 21, 2017 at 02:34 GMT_

----

Using `IEnumerable<>` allows you to provide multiple values (e.g. `-o a/b/c d/e/f g/h/i`) but no, it doesn't currently support providing `-o` multiple times.

**Comment by moh-hassan** [comment]
implemented in [v2.9.0-preview1](https://www.nuget.org/packages/CommandLineParser/)
